examples/HistogramExample.py

```diff
@@ -0,0 +1,135 @@
+# This is a simple system monitor that shows you CPU load histogram among other usefull stuff
+from luma.core.interface.serial import i2c
+from luma.core.render import canvas
+from luma.oled.device import ssd1306
+from datetime import timedelta
+import os
+import multiprocessing
+import time
+import psutil
+
+# Setting up the screen
+serial = i2c(port=0, address=0x3C)
+device = ssd1306(serial)
+
+# Longer refresh rate the more history is shown
+# If you set this to e.x. 30 sec you will get about 25 minutes of history on the graph
+refresh = 1
+
+# FlipFlop blink variable
+blnk = 1
+
+# HistogramSettings
+histogramResolution = 100
+histogramTime = []
+histogramData = []
+x = 106
+# Filling up the arrays for the histogram
+for pix in range(0, histogramResolution):
+    x -= 2
+    if x > 2:
+        histogramTime.append(x)
+
+for timeLen in range(0, len(histogramTime)):
+    histogramData.append(60)
+
+
+def main():
+    # Importing some global vars
+    global blnk
+    global histogramData
+    global histogramTime
+    # Starting the canvas for the screen
+    with canvas(device, dither=True) as draw:
+        # Vars:
+        # Getting system uptime
+        sysUptime = timedelta(seconds=float(open('/proc/uptime', 'r').readline().split()[0]))
```

Comment:
`/proc/uptime` should be closed after reading the file. use `with open(` etc instead.

Reply:
Oh, sorry for that.
I've only been typing python for a few days, will take a note on this

Reply:
no worries!

Reply:
Fixed - used `psutil.boot_time()` instead